src/index.html

```diff
@@ -33,17 +28,30 @@
             </div>
         </div>
 
-        <footer class="row fittable-footer">
-          <div class="build-info">
-            &copy; 2015&emsp;&emsp;FIT ČVUT v&nbsp;Praze
+        <footer class="row footer">
+          <div class="footer-usermenu">
+              <!-- fixme: missing translation, waiting for fittable+widget merge -->
+              <span class="footer-usermenu-loggedas">Přihlášen jako</span>
+              <span class="footer-usermenu-username">
+                <i class="fa fa-user"></i> <span id="username"></span>
+              </span>
+              <a href="_oauth/login" class="footer-usermenu-logout">
+                <i class="fa fa-power-off"></i>
+                <span>Odhlásit se</span>
+              </a>
           </div>
-          <ul>
-            <li><a href="https://github.com/cvut/fittable-widget"><i class="fa fa-github"></i>Fittable <span style="color: #cd4d4d;font-weight:bold;">BETA</span></a></li>
-            <li><a href="https://github.com/cvut/sirius/wiki">Sirius</a></li>
-            <li><a href="mailto:helpdesk@fit.cvut.cz">Report bug</a></li>
-          </ul>
-          <div style="padding-top: 2em; text-align: center;">
-            Narazili jste na chybu? Nahlašte nám ji prosím na <a href="mailto:helpdesk@fit.cvut.cz">helpdesk</a> a my ji co nejrychleji opravíme. Než se tak stane, můžete zatím využít <a href="https://timetable.fit.cvut.cz/old/">původní timetable</a>
+          <div class="footer-content">
+            <div class="footer-content-buildinfo">
+              &copy; 2015&emsp;&emsp;FIT ČVUT v&nbsp;Praze
+            </div>
+            <ul>
+              <li><a href="https://github.com/cvut/fittable-widget"><i class="fa fa-github"></i>Fittable <span style="color: #cd4d4d;font-weight:bold;">BETA</span></a></li>
+              <li><a href="https://github.com/cvut/sirius/wiki">Sirius</a></li>
+              <li><a href="mailto:helpdesk@fit.cvut.cz">Report bug</a></li>
+            </ul>
+            <div style="padding-top: 2em; text-align: center;">
+              Narazili jste na chybu? Nahlašte nám ji prosím na <a href="mailto:helpdesk@fit.cvut.cz">helpdesk</a> a my ji co nejrychleji opravíme. Než se tak stane, můžete zatím využít <a href="https://timetable.fit.cvut.cz/old/">původní timetable</a>
```

Comment:
Come on, ffs, I haven't created the pullrq. It's not ready for merging with master.